decidim-core/lib/decidim/core.rb

```diff
@@ -151,6 +152,18 @@ def self.seed!
     end
   end
 
+  def resource_reference_generator=(block)
+    warn("DEPRECATION WARNING: Decidim.resource_reference_generator is deprecated and will be removed in v0.10. Please use Decidim.reference_generator instead.")
+    @resource_reference_generator = block
```

Comment:
But wouldn't you need to update `@resource_generator` so that this actually works?

Reply:
Or call `resource_generator=`?